build/preprocess_template.py

```diff
@@ -70,7 +104,11 @@ def process_common_templates(commonTemplatesPath, rules):
     logging.info("Running script for file: " + fileName)
     logging.info("------------------------------------------------------")
 
-    updatedCommonTemplates = process_annotations(commonTemplates, rules)
+    updatedCommonTemplates = None
+    try:
+      updatedCommonTemplates = process_rules(commonTemplates, rules)
+    except Exception as e:
+      raise(e)
```

Comment:
You use this pattern already, but re-raising the exception we catch without any action in between is just useless.
We can fix this later however.

Reply:
I agree.

Reply:
i will update it